src/utils/syntax-highlight.js

```diff
@@ -192,7 +192,7 @@ function duplicateMultilineNode(element) {
 
   // wrap each new line with the current element's class
   const result = getLines(element.innerHTML)
-    .reduce((all, lineText) => `${all}<span class="${className}">${lineText || '\n\n'}</span>\n`, '');
+    .reduce((all, lineText) => `${all}<span class="${className}">${lineText || ''}</span>\n`, '');
```

Comment:
Hm. This change likely fixes one bug but it unfortunately re-introduces another that we previously had. These newlines were added to fix a bug where empty lines in a multiline Swift string were accidentally getting stripped.

With this code listing as an example:
```swift
let multiline = """
Needs

Spaces

Between

Lines
"""
```

On main, this gets rendered how it looks there.

On this branch, this gets rendered as follows:
```swift
let multiline = """
Needs
Spaces
Between
Lines
"""
```

Ideally we could find a solution that fixes #662 without re-introducing this other bug, although I know it's tricky because of the complex logic we have to handle line numbers and syntax highlighting.

Reply:
I think #502 was the PR for the bug fix that this change might accidentally re-introduce for context.

Reply:
No. This PR does not re-introduces the bug you mention. It passed the existing test and I tested this form and confirm it locally.

> Because the original use nothing. #502 add `|| "\n\n"`. But actually add `|| ""` is enough.

Reply:
Unfortunately it does re-introduce the issue:

In your docs, of you have this:

```swift
let multiline = """
Needs

Spaces

Between

Lines
"""
```

It will generate this:

![image](https://github.com/apple/swift-docc-render/assets/9863944/2849e816-c1e9-498f-becd-1a087171d4d6)

Reply:
> Unfortunately it does re-introduce the issue:
> 
> In your docs, of you have this:
> 
> ```swift
> let multiline = """
> Needs
> 
> Spaces
> 
> Between
> 
> Lines
> """
> ```
> 
> It will generate this:
> 
> ![image](https://user-images.githubusercontent.com/9863944/239461313-2849e816-c1e9-498f-becd-1a087171d4d6.png)

Got it. I thought it occur on the tutorial side.😂 And it was OK in tutorial steps.